src/js/components/RangeSelector/__tests__/__snapshots__/RangeSelector-test.js.snap

```diff
@@ -4553,6 +4553,7 @@ exports[`RangeSelector size 1`] = `
 >
   <div
     className="c1 c2"
+    size="xxsmall"
```

Comment:
This snapshot change indicates a potential regression in the DOM by the refactoring. the prop `size` should be part of the DOM div.